misc/gen-syntax.py

```diff
@@ -182,19 +183,25 @@ def pad_prefix(txt):
     return txt + (" " * (32 - len(txt)))
 
 
+def write_output(output, txt):
+    args.output.write(txt + "\n")
```

Comment:
It looks like this is supposed to be
```suggestion
    output.write(txt + "\n")
```
but I don't think this should send the output to the logger. If the user wants the output to be displayed, it can be sent to stdout. And in that case, we can write `print(line, file=output)` instead of having a `write_output` function.